FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib

```diff
@@ -281,6 +281,9 @@
         <namedColor name="custom_grey50">
             <color red="0.047058823529411764" green="0.047058823529411764" blue="0.050980392156862744" alpha="0.80000001192092896" colorSpace="custom" customColorSpace="sRGB"/>
         </namedColor>
+        <namedColor name="custom_grey50">
```

Comment:
Seems like a duplicated color definition?

Reply:
I've removed the duplicated color definition. Thank you!